gammapy/maps/tests/test_base.py

```diff
@@ -373,3 +373,40 @@ def test_map_arithmetics(map_type):
     m1 += 4
     assert m1.unit == u.Unit("")
     assert_allclose(m1.data, 4)
+
+
+def test_arithmetics_inconsistent_geom():
+    m_wcs = Map.create(binsz=0.1, width=1.0, map_type="wcs", skydir=(0, 0), unit="m2")
+    m_wcs_incorrect = Map.create(
+        binsz=0.1, width=2.0, map_type="wcs", skydir=(0, 0), unit="m2"
```

Comment:
Maybe remove some arguments here that are the default anyways, then the line break will be gone and overall a bit shorter and less code to read in the tests? (e.g. `skydir=(0, 0)` is the default, `map_type="wcs"` as well).

Reply:
Done